etc/onespan-sign-node.api.md

```diff
@@ -146,7 +217,43 @@ interface CreatePackage {
 }
 
 // @public
-export type CreatePackageData = PackageData;
+type CreatePackageData = PackageData;
+
+// @public (undocumented)
+type CreateSenderData = RecursivePartial<Sender>;
+
+// @public
+export type CustomData = Nullable<Record<string, any>>;
+
+// @public (undocumented)
+interface CustomField {
+    // (undocumented)
+    data: CustomData;
+    // (undocumented)
+    id: string;
+    // (undocumented)
+    name: string;
+    // (undocumented)
+    required: boolean;
+    // (undocumented)
+    translations: Translation[];
+    // (undocumented)
+    value: Nullable<string>;
+}
+
+// @public (undocumented)
+type Cycle = 'DAY' | 'MONTH' | 'YEAR';
+
+// @public (undocumented)
+interface CycleCount {
+    // (undocumented)
+    count: number;
+    // (undocumented)
+    cycle: Cycle;
+}
+
+// @public
+export type DateTimeString = string;
```

Comment:
```suggestion
export type DateTimeString = Date;
```

Maybe?

Reply:
`DateTimeString` is an alias of `string` with a specific format 
See: https://github.com/getbreathelife/onespan-sign-node/pull/26/files#diff-eb1583bd556654ba410ab49100070d7bb28ebfb156b1f813e00a858fc6689d8bR13-R16

In the future, we could perform more transformations on the responses but it's not in the plan right now